Resources/doc/basic-usage.rst

```diff
@@ -79,16 +79,18 @@ If you need to access filtered image URL in your controller:
     
     $this->get('liip_imagine.cache.manager')->getBrowserPath('/relative/path/to/image.jpg', 'my_thumb', true),
 
-In this case, the final rendered path would contain some random data in the path
-`/media/cache/my_thumb/S8rrlhhQ/relative/path/to/image.jpg`. This is where Imagine
-would save the filtered image file.
+In this case, the final rendered path would contain some random data in the 
+path ``/media/cache/my_thumb/S8rrlhhQ/relative/path/to/image.jpg``. This is where 
+Imagine would save the filtered image file.
 
 .. note::
-    Using the ``dev`` environment you might find that the images are not properly rendered when
-    using the template helper. This is likely caused by having ``intercept_redirect`` enabled in your
-    application configuration. To ensure that the images are rendered disable this option:
-
-    .. code-block:: jinja
-        
-        web_profiler:
-            intercept_redirects: false
+
+    Using the ``dev`` environment you might find that the images are not properly 
+    rendered when using the template helper. This is likely caused by having `
+    `intercept_redirect`` enabled in your application configuration. To ensure 
+    that the images are rendered disable this option:
+
+.. code-block:: yaml
```

Comment:
this should be indented, as it's part of the note directive (just like you did before).